### PR TITLE
Graphormer fix 

### PR DESCRIPTION
**Pull request by clefourrier**
Removes failing call to `requires_backend` in Graphormer model since the model uses `is_cython_available` instead.

@ydshieh

### CODE REVIEW

File: src/transformers/models/graphormer/collating_graphormer.py [hunk 1]
```diff
@@ -24,7 +24,6 @@ def convert_to_single_emb(x, offset: int = 512):
 
 
 def preprocess_item(item, keep_features=True):
-    requires_backends(preprocess_item, ["Cython"])
```
Comment (ydshieh) [inline src/transformers/models/graphormer/collating_graphormer.py:27]: I remembered that, from another PR,  Sylvain mentions we prefer using `requires_backends`. @amyeroberts Is my memory correct?

Comment (amyeroberts) [reply src/transformers/models/graphormer/collating_graphormer.py:27]: That's my understanding too. It should be equivalent to `is_cython_available` and be a safe check i.e. shouldn't cause a failure other than raising an exception if not available. 

@clefourrier could you give more details about failure?

Comment (clefourrier) [reply src/transformers/models/graphormer/collating_graphormer.py:27]: Very likely my fault, coming from when I implemented the checks for Cython

```python
File ~/Desktop/huggingface_env/lib/python3.8/site-packages/transformers/utils/import_utils.py:1035, in <genexpr>(.0)
   1032 if "tf" in backends and "torch" not in backends and is_torch_available() and not is_tf_available():
   1033     raise ImportError(TF_IMPORT_ERROR_WITH_PYTORCH.format(name))
-> 1035 checks = (BACKENDS_MAPPING[backend] for backend in backends)
   1036 failed = [msg.format(name) for available, msg in checks if not available()]
   1037 if failed:

KeyError: 'Cython'
```

Comment (clefourrier) [reply src/transformers/models/graphormer/collating_graphormer.py:27]: Solved this, thank you very much for your tips!

**Review by ydshieh — APPROVED** [review]
LGTM, but let's have a core maintainer's review too 🙏 
Thank you for the PR, @clefourrier !

**Review by amyeroberts — APPROVED** [review]
LGTM - thanks for adding this!

**Review by sgugger — APPROVED** [review]
Thanks for the fix!

**Pull request merged**